app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileResetPinActivityTest.kt

```diff
@@ -120,7 +120,7 @@ class ProfileResetPinActivityTest {
   }
 
   @Test
-  fun testProfileResetPinActivity_startActivityWithAdmin_inputPin_inputConfirmPin_clickSave_checkReturnsToProfileEditActivity() { // ktlint-disable max-line-length
+  fun testResetPin_asAdmin_inputPinAndConfirmPin_clickSave_checkReturnsToProfileEdit() {
```

Comment:
please keep the prefix unchanged `testProfileResetPinActivity`

Reply:
Let me know if we can do without changing prefix

Reply:
@anandwana001 Do you mean keeping the prefix unchanged on all test function names or only this specific one (or all short ones)? I'll try without changing the prefix, although it will reduce the ease of understanding of the rest of the function.

Reply:
Short ones it will be easy enough to keep the prefix, but there's some functions which are already at the 100 character mark which will be difficult to change since they also have `changeConfig` in them.

Reply:
Let's go with `testProfileResetPin`

Reply:
Marking this as resolved.